content/pt-br/docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configurando a Inicialização do Pod
+content_type: task
+weight: 130
+reviewers:
+---
+
+<!-- overview -->
+Esta página mostra como usar um contêiner de incialização para um Pod, que será executado antes de um 
+aplicativo ser iniciado.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Crie um Pod que tem um Contêiner de incialização
+
+Neste exercício, você cria um pod que possui um aplicativo de contêiner e um
+contêiner de inicialização. O contêiner de incialização é executado até a sua conclusão, antes da aplicação
+do contêiner ser iniciada.
+
+Aqui está o arquivo de configuração do pod:
+
+{{< codenew file="pods/init-containers.yaml" >}}
+
+No arquivo de configuração, você pode ver que o Pod tem um volume 
+que o contêiner de inicialiazação e o contêiner da aplicação compartilham.
+
+O contêiner de incialização monta o volume compartilhado em `/work-dir`, e o contêiner 
+da aplicação monta o volume compartilhado em `/usr/share/nginx/html`. 
+O contêiner de incialização executa o seguinte comando e depois termina:
+
+    wget -O /work-dir/index.html http://info.cern.ch
```

Comment:
````suggestion
```shell
wget -O /work-dir/index.html http://info.cern.ch
```
````

Using the shell fenced code block renders the code a little differently from the other code blocks.